### PR TITLE
[post] GUI Challenges - SVG Favicon

### LINKED ISSUE #7250: [CSS] Building an adaptive favicon [2/9/2022]
* Proposed title: Building an adaptive favicon
* Proposed by: argyle
* Related areas: CSS
* Content type: How-To/API Guide (https://documentation.divio.com/how-to-guides/)
* Deadline: 2/9/2022
* Deadline explanation: Land with the associated YouTube video 
* [Draft](https://docs.google.com/document/d/1YnDLfD_SByEzpJZCojpAovB0-qdwY15nUl73a3-GmXw/edit#)
* Content summary: manage less icons and deliver higher quality + theme adapting colors. also has a great progressive enhancement path.

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #7250

ToDo:
- [x] final content reviews
- [x] add final youtube embed key

### CODE REVIEW

File: src/site/content/en/blog/building-an-adaptive-favicon/index.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+also inside "save for later" readers, other blog posts linking to your site, and
+more. Traditionally this has been done with the `.ico` file type, but recently
+browsers have allowed use of
+[SVG](https://developer.mozilla.org/docs/Web/SVG), a vector format. Using
+[progressive
+enhancement](https://developer.mozilla.org/docs/Glossary/Progressive_Enhancement)
+you can  serve well supported `.ico` favicons, and upgrade to an `.svg` if
+available.
+
+{% BrowserCompat 'api.linkIconSVG' %}
```
Comment (argyleink) [inline src/site/content/en/blog/building-an-adaptive-favicon/index.md:50]: i can find the caniuse for SVG favicons but struggled to find the api key for it for use here..

Comment (argyleink) [reply src/site/content/en/blog/building-an-adaptive-favicon/index.md:50]: is there a good way to find the ID from [here](https://caniuse.com/link-icon-svg)?

Comment (rachelandrew) [reply src/site/content/en/blog/building-an-adaptive-favicon/index.md:50]: I replied about this in chat but it isn't in BCD data.

File: src/site/content/en/blog/building-an-adaptive-favicon/index.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+[SVG is able](https://css-tricks.com/lodge/svg/) to scale up and down without
+quality loss, and can potentially be very small in size, they can also have
+embedded CSS, even embedded media queries. This means if an SVG favicon is used
+in a reader app or bookmarks bars, there's a chance the user could get a theme
+relevant (light or dark) icon due to dark preference styles provided inside the
+SVG. The SVG then adapts using its embedded styling for light and dark user
+preferences.
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/c8pUK6HOlCuehgtqzFci.png",
+alt="Large and easy to distinguish light and dark favicon examples",
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-an-adaptive-favicon/index.md:61]:
```suggestion
alt="Large and easy to distinguish light and dark favicon examples.",
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**